queries.yaml

```diff
@@ -203,3 +203,47 @@ pg_stat_statements:
     - blk_write_time_seconds:
         usage: "COUNTER"
         description: "Total time the statement spent writing blocks, in milliseconds (if track_io_timing is enabled, otherwise zero)"
+
+pg_stat_activity:
```

Comment:
Note that this is overriding the built-in pg_stat_activity metrics, is that expected?